src/index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,400;0,9..40,500;0,9..40,600;0,9..40,700;0,9..40,800;1,9..40,400&family=Quicksand:wght@400;500;600;700&display=swap" rel="stylesheet">
   </head>
-  <body class="grid min-h-screen">
+  <body class="grid min-h-dvh">
```

Comment:
[`dvh`](https://tailwindcss.com/blog/tailwindcss-v3-4#dynamic-viewport-units) is shipped in tailwind 3.4

Reply:
That's a useful class. :+1: